library/L2_HAL/sensors/optical/apds9960.hpp

```diff
@@ -319,18 +320,13 @@ class Apds9960
     overflow_value         = gesture_.memory.gesture_status;
     if (overflow_value & 0b10)  // if overflow, clear FIFO data
```

Comment:
```suggestion
    if (overflow_value & 0b10)  // if overflow, clear FIFO data
```
Use bit:: API and use constants.